docker/Dockerfile.higress

```diff
@@ -13,7 +13,12 @@ FROM ${HUB}/base:${BASE_VERSION}
 ARG TARGETARCH
 
 COPY ${TARGETARCH:-amd64}/higress /usr/local/bin/higress
+COPY wait-for-apiserver.sh /usr/local/bin/wait-for-apiserver.sh
+COPY run-higress.sh /usr/local/bin/run-higress.sh
+
+RUN chmod +x /usr/local/bin/wait-for-apiserver.sh
+RUN chmod +x /usr/local/bin/run-higress.sh
```

Comment:
Does one "RUN" statement here generate one layer in the image? Could we run both commands in one "RUN" statement, or just `RUN chmod +x /usr/local/bin/*.sh`?

Reply:
Maybe I can put these into `run-higress.sh`.